files/tasks.py

```
@@ -1246,48 +1273,119 @@ def beat_test(x, y):
+        logger.error(f"Error finding ffmpeg processes for {filepath}: {e}")
     return None
 
 
+def kill_all_ffmpeg_processes():
+    """Kill ALL ffmpeg processes on this worker.
+
+    Used during worker shutdown to ensure no orphan ffmpeg processes remain.
+    """
+    try:
+        result = subprocess.run(
+            ["pgrep", "-f", "ffmpeg"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)